test/fixtures/user.ts

```diff
@@ -0,0 +1,13 @@
+import { User } from "../../src/modules/user/model/user.entity";
+
+export function mockUserEntity(overrides?: Partial<User>): User {
+  return {
+    id: Math.floor(Date.now() / 1000),
```

Comment:
Is it necessary to mock the `id` parameter? It should be created by the database automatically

Reply:
Yea the id gets overridden by the db but sometimes I need to mock the whole `User` entity without saving it in the db. For example for test cases with non-existing users.